speculos/main.py

```diff
@@ -433,8 +439,12 @@ def main(prog=None):
     if api_enabled:
         apirun = ApiRunner(args.api_port)
 
+    if disable_tesseract: 
+        args.disable_tesseract = True
+
     display_args = display.DisplayArgs(args.color, args.model, args.ontop, rendering,
-                                       args.keymap, zoom, x, y, args.force_full_ocr)
+                                       args.keymap, zoom, x, y, args.force_full_ocr,
+                                       args.disable_tesseract)
```

Comment:
Why not directly using `disable_tesseract` here? You don't need to set it in args I think?

Reply:
my bad, I didn't see that you added an option for that...